### PR TITLE
completed mergeArrays exercise

### PR DESCRIPTION
**Pull request by goelvatsal**

### CODE REVIEW

File: section6Arrays/mergeArrays/main.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package mergeArrays
+
+import "sort"
+
+func mergeArrays(slc1, slc2 []int) []int {
+	var mergeSlice []int
+
+	mergeSlice = append(mergeSlice, slc1...)
+	mergeSlice = append(mergeSlice, slc2...)
```
Comment (goel4607) [inline section6Arrays/mergeArrays/main.go]: I think you should write your own merge logic rather than using library method

Comment (goelvatsal) [reply section6Arrays/mergeArrays/main.go]: fixed

File: section6Arrays/mergeArrays/main.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package merge_arrays
```
Comment (goel4607) [inline section6Arrays/mergeArrays/main.go]: pls rename as suggested in other PR 
Also rename the file from main to something more meaningful.
Introduce an interface here. Lets discuss, when you have time.

Comment (goelvatsal) [reply section6Arrays/mergeArrays/main.go]: Ok

**Review by goel4607 — APPROVED** [review]

**Pull request merged**